src/osal/lv_freertos.c

```diff
@@ -74,9 +74,13 @@ static void prvTestAndDecrement(lv_thread_sync_t * pxCond,
 #if (ESP_PLATFORM)
     #define _enter_critical()   taskENTER_CRITICAL(&critSectionMux);
     #define _exit_critical()    taskEXIT_CRITICAL(&critSectionMux);
+    #define _enter_critical_isr() _enter_critical()
+    #define _exit_critical_isr()  _exit_critical()
```

Comment:
```suggestion
    #define _enter_critical_isr() taskENTER_CRITICAL_FROM_ISR()
    #define _exit_critical_isr(x)  taskEXIT_CRITICAL_FROM_ISR(x)
```

I tested on ESP32 and that API is available. I think it would be good to use it here too.

Reply:
It has to be this or FreeRTOS will assert in `vPortExitCritical` - at least on the CM4/7 port